README.md

```diff
@@ -1,11 +1,11 @@
 # WP-Droid [WIP]
 
-A project to convert your Wordpress website into an android app. This app uses [WP API](v2.wp-api.org/) to work properly.
-The app is just an implementation of how wordpress api can be used to extend wordpress api and convert blog into an app.
+A project to convert your wordpress website into an android app. This app uses the [WP API](v2.wp-api.org/) to work properly.
+The app is just an implementation of how WordPress API can be used to extend WordPress API and convert the blog into an app.
```

Comment:
Looking back, this sentence seems incoherent. Maybe we can refactor it to
```
The repo showcases how to convert your existing Wordpress blog into an android app using the Wordpress API.
```

Reply:
We can change it to 
"The app is just an implementation of the way WordPress API extends its own API to convert the blog into an app."